package.json

```diff
@@ -36,6 +36,8 @@
     "phantomjs": "^1.9.17",
     "promisehelpers": "0.0.3",
     "q": "^1.3.0",
+    "sinon": "^1.15.3",
+    "sinon-chai": "^2.8.0",
     "superagent": "^1.2.0"
   },
   "peerDependencies": {
```

Comment:
Can you also remove peerDependencies and add q and superagent to dependencies, both at `1.x.x`?